src/styles/Css/App.css

```diff
@@ -21,44 +20,46 @@ body {
 body h2,
 body span,
 body h3 {
-  color: #03045e;
+  color: #121212;
```

Comment:
- [ ] Again here it would be better to use a reusable class for this black color as it has been used several times